src/webviews/apps/settings/partials/commit-graph.html

```diff
@@ -170,6 +170,20 @@ <h2>
 						</div>
 					</div>
 
+					<div class="setting">
+						<div class="setting__input">
+							<input
+								id="graph.pullRequests.enabled"
+								name="graph.pullRequests.enabled"
+								type="checkbox"
+								data-setting
+							/>
+							<label for="graph.pullRequests.enabled"
+								>Show associated pull requests on remote branches</label
+							>
+						</div>
+					</div>
```

Comment:
Add
```html
<p class="setting__hint hidden" data-visibility="currentLine.pullRequests.enabled">
	<i class="icon icon__info"></i>Requires a connection to a supported remote service (e.g. GitHub)
</p>
```

Reply:
Thanks. Should I add that for the upstream indicator setting as well?

Reply:
Done in commit: [Add hints for remote service connection](https://github.com/gitkraken/vscode-gitlens/pull/2450/commits/19b2f1435bdabc519efb051bdbbbfc57fe267fc9)

Reply:
Upstream, doesn't need an integration -- just the PRs

Reply:
Got it, thanks! Fixed in commit: [Remove remote service hint from upstream setting](https://github.com/gitkraken/vscode-gitlens/pull/2450/commits/30826595701a68a8e031799236abfc60174f826f)